aiven/client/argx.py

```diff
@@ -124,7 +107,7 @@ def __setattr__(self, name: str, value: Callable) -> None:
 arg = Arg()
 
 
-def name_to_cmd_parts(name: str) -> List[str]:
+def name_to_cmd_parts(name: str) -> list[str]:
```

Comment:
`list[T]` is only available in 3.9+

Reply:
Starting with Python 3.7, when from __future__ import annotations is used, function and variable annotations can parameterize standard collections directly.

From:
https://peps.python.org/pep-0585/#implementation

Reply:
https://github.com/python/mypy/issues/7907